server/server.js

```diff
@@ -16,8 +16,30 @@ app.use(express.json());
 
 app.get('/', (req, res) => {
     res.json('nuevo comienzo')
-})
+});
+
+app.get('/users', async (req, res) => {
+    try {
+        const { rows: users } = await db.query('SELECT * FROM users');
+        res.send(users);
+    } catch (error) {
+        console.error('Error in Database Query:', error);
+        return res.status(400).json({ error });
```

Comment:
I'm not sure about returning a 400 error all the time here. It can be 400 (client side, there's a problem with your request), or 500 (your request was fine, but there's a problem with the server). Maybe an optimization and further reading for later.

Reply:
Hmm...this is very insightful.  Thank you for sharing.  I changed them to 500, and made a not in the client side to use 400 there for errors.